src/web/components/Callout/FieldLabel.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { textSans } from '@guardian/src-foundations/typography';
+import { neutral } from '@guardian/src-foundations/palette';
+
+const fieldLabelStyles = css`
+    ${textSans.medium({ fontWeight: 'bold' })}
+`;
+
+const fieldDescription = css`
+    ${textSans.medium()}
+`;
+
+const optionalTextStyles = css`
+    ${textSans.small({ fontStyle: 'italic' })}
+    color: ${neutral[46]};
+    padding-left: 5px;
+`;
+
+export const FieldLabel = ({ formField }: { formField: CampaignField }) => (
+    <label className={fieldLabelStyles} htmlFor={formField.name}>
+        <div>
+            {formField.label}
+            {formField.required && (
```

Comment:
Is this logic a bit reversed? I would expect the optional text to only show if the field is _not_ required

Reply:
good catch! yes previously this was comparing a string '0' and I forgot to inverse the bool